app/presenters/export/csv/project_presenter.rb

```diff
@@ -182,4 +182,32 @@ def trust_surplus_deficit
     return I18n.t("export.csv.project.values.not_applicable") if @project.tasks_data.check_and_confirm_financial_information_not_applicable?
     @project.tasks_data.check_and_confirm_financial_information_trust_surplus_deficit
   end
+
+  def diocese_name
+    @project.establishment.diocese_name
+  end
+
+  def diocese_contact
+    @project.contacts.where(category: "diocese")
```

Comment:
This will return a Contact record, can you return the email address (I presume) of this record. Plus guard for if the record doesn't exist.